### DISCUSSION TITLE
how to redirect link in news to project section?

### DISCUSSION THREAD

**Discussion by SamuelMlu** [alshedivat/al-folio#854] [Q&A] [answered]
I want to make the news column in the about pages become a page that displays changes to my page. is it possible when I make a link in the news then it is redirected to my project page?

**Comment by cmoralesmx** [comment, marked as answer]
Links can be directed to anything internal or external. Here's an example of a news entry with a relative link to the projects page

```markdown
---
layout: post
title: title
date: 2022-10-08 17:11:00-0400
inline: true
---

A news entry linking to the [projects](projects) page
```

**Reply by SamuelMlu** [reply]
Thank you
